Release notes update for 5.3.1

tst-nfarah opened the pull request:
This address #1365 

Edited document can be viewed [here](https://tst-nfarah.github.io/earthenterprise/geedocs/5.3.1/answer/7160008.html).

tst-eclamar commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -73,13 +71,41 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
```
When I (mistakenly) did this, I found a few more: 393, 1236, 1277, 1292, 1300, & 1301.  Are these not valid?

tst-nfarah replied:
Good catch.
393 --> It was fixed in master: 5.3.2
1236, 1239: Ok
1277: OK
1292: OK
1300: Internal clean up
1301: OK

I have updated the release notes adding the issues i tagged above as OK.

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -40,8 +40,6 @@
-<p>Open GEE 5.3.1 is currently in development</p>
-
   <h5>
     <p id="overview_5.3.1">New Features</p>
```
Can we make these section headers larger? Maybe h3 or h4.

tst-nfarah replied:
ok

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -40,8 +40,6 @@
-
   <h5>
     <p id="overview_5.3.1">New Features</p>
   </h5>
```
In line 49, below change "has new option" to "have a new option"

tst-nfarah replied:
yep

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -73,13 +71,69 @@ <h5>
             <th>Resolution</th>
           </tr>
+          <tr>
+            <td>1236</td>
```
This seems too low level to include, in my opinion

tst-nfarah replied:
I will remove the listed fixed issues that are low level code changes, since they don't appear to yield direct user benefit.

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -73,13 +71,69 @@ <h5>
+            </td>
+          </tr>
+          <tr>
+            <td>1277</td>
```
Same comment as above. Either remove or change description to indicate how this helps end users.

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -73,13 +71,69 @@ <h5>
+            <td>1301</td>
+            <td>Combine storage-related code.</td>
+            <td>
+              This is a re-factoring work to create a storage manager that manages reading, writing, and caching.
```
Indicate how this benefits the end-user or remove

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -73,13 +71,69 @@ <h5>
+            <td>1344</td>
+            <td>No method in GlcUnpacker to "walk" a single layer</td>
+            <td>
+              Added method in GlcUnpacker to "walk" a single layer.
```
I don't even understand what this means. Please clarify from an end-user's perspective

tst-nfarah commented:
> Several comments. We need to start thinking about these release notes from an end-user's perspective.

@tst-ccamp I addressed your review comments.

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -40,18 +40,16 @@
-
   <h5>
-    <p id="overview_5.3.1">New Features</p>
+    <h3 id="overview_5.3.1">New Features</h3>
```
I don't think it's a good idea to have h3 inside an h5 block. You can probably just remove the h5 here

tst-nfarah replied:
done

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -62,30 +57,54 @@ <h5>
-    <h5>
-      <p id="resolved_issues_5.3.1">Resolved Issues</p>
-    </h5>
+    <h3 id="resolved_issues_5.3.1">Resolved Issues</h3>
```
There is still an h3 tag inside an h5 tag. Please fix as above

tst-nfarah replied:
Removed

tst-ccamp commented on docs/geedocs/5.3.1/answer/7160008.html:
```diff
@@ -62,30 +57,54 @@ <h5>
-  <h5>
-    <p id="known_issues_5.3.1">Known Issues</p>
-  </h5>
+    <h3 id="known_issues_5.3.1">Known Issues</h3>
```
Please fix this as well

tst-nfarah replied:
Done

tst-nfarah commented:
@tst-ccamp I addressed your latest comments.
